3x-ui/umbrel-app.yml

```diff
@@ -0,0 +1,53 @@
+manifestVersion: 1
+id: 3x-ui
+name: 3x-UI
+tagline: Xray panel supporting multi-protocol multi-user expire day & traffic & ip limit (Vmess & Vless & Trojan & ShadowSocks & Wireguard)
+category: networking
+version: "2.3.8"
+port: 2053
+description: >-
+  3X UI is an advanced web panel built on Xray Core, designed for managing various proxy protocols including Shadowsocks-2022, VLESS with XTLS, and more. This user-friendly interface simplifies the management of Xray servers, providing detailed insights and powerful features.
+  
+  ☺️ Simple & Easy-to-use
+  
+  3X UI offers a straightforward setup process and handles server management efficiently. The interface is intuitive, making it accessible even for users with minimal technical knowledge.
+  
+  🔐 Cryptographically Sound
+  
+  Built on Xray Core, 3X UI utilizes state-of-the-art cryptographic protocols to ensure secure and reliable connections. It supports multiple protocols, including VMess, VLESS, Trojan, and Shadowsocks.
+  
+  🛡️ Minimal Attack Surface
+  
+  Designed with security in mind, 3X UI has a streamlined codebase that minimizes potential vulnerabilities. This makes it easier to audit and maintain.
+  
+  ⏱️ High Performance
+  
+  Thanks to efficient cryptographic primitives and optimization, 3X UI ensures fast and secure networking, capable of handling high traffic loads with ease.
+  
+  📐 Well Defined & Thoroughly Considered
+  
+  3X UI is meticulously designed with a focus on reliability and performance. Its robust feature set includes traffic statistics, user management, and custom configuration templates.
+
+  🎨 Dark/Light Theme Support
+  
+  Easily switch between dark and light themes to match your preference and reduce eye strain during extended use.
+
+  🛠️ Extensive Configuration Options
+  
+  Supports HTTPS access panel, one-click SSL certificate management, and advanced configuration options for customizing your server setup.
```

Comment:
Awesome description  :D
One minor suggestion: Add a blank line in-between the paragraphs. Two blank lines here equal one blank line in the UI. One blank line here equals a line break.
```suggestion
3X UI is an advanced web panel built on Xray Core, designed for managing various proxy protocols including Shadowsocks-2022, VLESS with XTLS, and more. This user-friendly interface simplifies the management of Xray servers, providing detailed insights and powerful features.


☺️ Simple & Easy-to-use
  
3X UI offers a straightforward setup process and handles server management efficiently. The interface is intuitive, making it accessible even for users with minimal technical knowledge.


🔐 Cryptographically Sound

Built on Xray Core, 3X UI utilizes state-of-the-art cryptographic protocols to ensure secure and reliable connections. It supports multiple protocols, including VMess, VLESS, Trojan, and Shadowsocks.


🛡️ Minimal Attack Surface

Designed with security in mind, 3X UI has a streamlined codebase that minimizes potential vulnerabilities. This makes it easier to audit and maintain.


⏱️ High Performance

Thanks to efficient cryptographic primitives and optimization, 3X UI ensures fast and secure networking, capable of handling high traffic loads with ease.


📐 Well Defined & Thoroughly Considered

3X UI is meticulously designed with a focus on reliability and performance. Its robust feature set includes traffic statistics, user management, and custom configuration templates.


🎨 Dark/Light Theme Support

Easily switch between dark and light themes to match your preference and reduce eye strain during extended use.


🛠️ Extensive Configuration Options

Supports HTTPS access panel, one-click SSL certificate management, and advanced configuration options for customizing your server setup.
```